packages/block-editor/src/utils/transform-styles/transforms/url-rewrite.js

```diff
@@ -1,138 +1,33 @@
 /**
- * Return `true` if the given path is http/https.
- *
- * @param  {string}  filePath path
- *
- * @return {boolean} is remote path.
+ * External dependencies
  */
-function isRemotePath( filePath ) {
-	return /^(?:https?:)?\/\//.test( filePath );
-}
+import { URL } from 'whatwg-url';
```

Comment:
I don't think this dependency is necessary anymore. `URL` has achieved a good level of standardization.

Reply:
Yes, good point. Browser support is basically 100% now. NodeJS supports it, too.